packages/rx_bloc_cli/example/README.md

```diff
@@ -64,7 +65,7 @@ Before you start working on your app, make sure you familiarize yourself with th
 | `lib/lib_router/routes`                      | Declarations of all nested pages in the application are located here                                                                                  |  
 | `lib/lib_dev_menu`                           | A useful feature when it comes to debugging your app by easily set and access proxy debugging services Charles and Alice.                             |
 | `lib/feature_otp`                            | Contains a number of useful widgets that can help you with building sms/pin code screens or workflows for your app.                                   |
-
+| `lib/lib_pin_code`    
```

Comment:
Add the description and finish the table

Reply:
The description was added